docs/data/sdks/android-kotlin/index.md

```diff
@@ -293,3 +293,494 @@ amplitude.add(
 )
 ```
+
+## Advanced Topic
+
+### User sessions
+
+A session on Android is a period of time that a user has the app in the foreground.
+
+Amplitude groups events together by session. Events that are logged within the same session have the same `session_id`. Sessions are handled automatically so you don't have to manually call `startSession()` or `endSession()`.
+
+You can adjust the time window for which sessions are extended.
+
+=== "Java"
+
+    ```
+    amplitude = AmplitudeKt.Amplitude(AMPLITUDE_API_KEY, getApplicationContext(), configuration -> {
+        configuration.setMinTimeBetweenSessionsMillis(1000);
+        return Unit.INSTANCE;
+    });
+    ```
+=== "Kotlin"
+
+    ```kotlin
+    amplitude = Amplitude(
+            Configuration(
+                apiKey = AMPLITUDE_API_KEY,
+                context = applicationContext,
+                minTimeBetweenSessionsMillis = 10000
+            )
+        )
+    ```
+
+By default, Amplitude will auto send the '[Amplitude] Start Session' and '[Amplitude] End Session'. Even though these events aren't sent, sessions are still tracked by using `session_id`.
+You can also disable those session events.
+
+=== "Java"
+
+    ```
+    amplitude = AmplitudeKt.Amplitude(AMPLITUDE_API_KEY, getApplicationContext(), configuration -> {
+        configuration.setTrackingSessionEvents(false);
+        return Unit.INSTANCE;
+    });
+    ```
+=== "Kotlin"
+
+    ```kotlin
+    amplitude = Amplitude(
+            Configuration(
+                apiKey = AMPLITUDE_API_KEY,
+                context = applicationContext,
+                trackingSessionEvents = false
+            )
+        )
+    ```
+
+You can use the helper method getSessionId to get the value of the current sessionId.
+
+=== "Java"
+
+    ```java
+    long sessionId = amplitude.getSessionId();
+    ```
+
+=== "Kotlin"
+
+    ```kotlin
+    val sessionId = amplitude.sessionId;
+    ```
+
+!!!note
+    For Android API level 14 and above, a new session is created when the app comes back into the foreground after being in the background for five or more minutes or when the last event was logged
+     (whichever occurred last).
+     Otherwise, the background event logged is part of the current session.
+      Note that you can define your own session expiration time by calling setMinTimeBetweenSessionsMillis(timeout), where the timeout input is in milliseconds.
+
+    For Android API level 13 and below, foreground tracking is not available so a new session is automatically started when an event is logged 30 minutes or more after the last logged event. If another event is logged within 30 minutes, it will extend the current session. Note that you can define your own session expiration time here as well by calling setSessionTimeoutMillis(timeout), where the timeout input is in milliseconds. Also note that enableForegroundTracking(getApplication) is still safe to call for Android API level 13 and below even though it's not available.
```

Comment:
```suggestion
    For Android API level 13 and lower, foreground tracking is not available so a new session is automatically started when an event is logged 30 minutes or more after the last logged event. If another event is logged within 30 minutes, it extends the current session. You can define your own session expiration time by calling `setSessionTimeoutMillis(timeout)`, where the timeout input is in milliseconds. It's safe to call `enableForegroundTracking(getApplication)` for Android API level 13 and lower even though it's not available.
```

Is "Android API level" the correct term, or should we be using "version"?

Reply:
Yes, it should be "Android API Level".